[2022-08-03 15:30:11,795] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T15:20:00+00:00 [queued]>
[2022-08-03 15:30:11,849] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: my_simple_dag.respond 2022-08-03T15:20:00+00:00 [queued]>
[2022-08-03 15:30:11,850] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:30:11,851] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-03 15:30:11,852] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:30:11,876] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-03T15:20:00+00:00
[2022-08-03 15:30:11,881] {standard_task_runner.py:51} INFO - Started process 30681 to run task
[2022-08-03 15:30:11,888] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'my_simple_dag', 'respond', '2022-08-03T15:20:00+00:00', '--job-id', '1224', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_simple_dag.py', '--cfg-path', '/tmp/tmpjrh5e28_']
[2022-08-03 15:30:11,893] {standard_task_runner.py:76} INFO - Job 1224: Subtask respond
[2022-08-03 15:30:11,975] {logging_mixin.py:103} INFO - Running <TaskInstance: my_simple_dag.respond 2022-08-03T15:20:00+00:00 [running]> on host b4dcda255553
[2022-08-03 15:30:12,117] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=my_simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T15:20:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-03T15:20:00+00:00
[2022-08-03 15:30:12,121] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-03 15:30:12,168] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=my_simple_dag, task_id=respond, execution_date=20220803T152000, start_date=20220803T153011, end_date=20220803T153012
[2022-08-03 15:30:12,214] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-03 15:30:12,261] {local_task_job.py:118} INFO - Task exited with return code 0
